iteration/iteration_test.go

```diff
@@ -207,7 +203,7 @@ func (s *TestIterationRepository) TestListIterationBySpace() {
 			SpaceID:    spaceInstance.ID,
 			StartAt:    &start,
 			EndAt:      &end,
-			UserActive: &userActive,
+			UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c